pipelines/resource_pipeline_test.go

```diff
@@ -474,7 +474,7 @@ func TestResourcePipelineDelete(t *testing.T) {
 		Fixtures: []qa.HTTPFixture{
 			{
 				Method:   "DELETE",
-				Resource: "/api/2.0/pipelines/abcd",
+				Resource: "/api/2.0/pipelines/abcd?",
```

Comment:
I'm not sure that we need to add `?` if query string is empty. 
I would say that this is a Go SDK problem